site/src/__mocks__/react-markdown.tsx

```diff
@@ -1,6 +1,6 @@
 import { FC } from "react"
 
-const ReactMarkdown: FC = ({ children }) => {
+const ReactMarkdown: FC<React.PropsWithChildren<unknown>> = ({ children }) => {
```

Comment:
Can we import `PropsWithChildren` directly from React?

Reply:
Ah, maybe you used a codemod. 
I assume we can also: 
````
const Component: FC<Props> = ({children}) => {...}
````
but maybe that wasn't an option?

Reply:
@Kira-Pilot `children` has to be explicit in the props now.


Also, I can import `PropsWithChildren` directly but it would be tedious for me to visit each file. I did use a codemod for the first pass.

If required for merge I will suck that up and make all the changes.

Reply:
No, not required - just curious

Reply:
Ok great :)